integration/src/main/resources/configs/integration/Jobs.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Jobs",
+  "description": "Test the JobsApi",
+  "serverSpecificationFile": "workspace-local.json",
+  "kubernetes": {},
+  "application": {},
+  "testScripts": [
+    {
+      "name": "Jobs",
+      "numberOfUserJourneyThreadsToRun": 1,
+      "userJourneyThreadPoolSize": 2,
```

Comment:
nit: this should also be 1, the thread pool size doesn't need to be larger than the number of threads to run